rootex/framework/components/hierarchy_component.cpp

```diff
@@ -25,6 +25,30 @@ bool HierarchyComponent::removeChild(Ref<Entity> node)
 	return true;
 }
 
+JSON::json HierarchyComponent::getJSON() const
+{
+	JSON::json j;
+
+	if (!m_Parent)
+	{
+		j["parent"]["ID"] = 1;
+		j["parent"]["name"] = "Root";	
+	}
+	else
+	{
+		j["parent"]["ID"] = m_Parent->getID();
+		j["parent"]["name"] = m_Parent->getName();
+	}
+
+	JSON::json& jc = j["children"];
```

Comment:
'jc' gives no idea about the variable. Rename it.

Reply:
'j' for JSON and 'c' for the 'children' field in that JSON